R_Scripts/expanded_spectra-trait_kit_lma_plsr_example.R

```diff
@@ -22,12 +22,18 @@ closeAllConnections()   # close any open connections to files
 
 #--------------------------------------------------------------------------------------------------#
 ### Install and load required R packages
-list.of.packages <- c("devtools","readr","RCurl","httr","pls","dplyr","reshape2","here",
+list.of.packages <- c("devtools","remotes","readr","RCurl","httr","pls","dplyr","reshape2","here",
                       "ggplot2","gridExtra")  # packages needed for script
 # check for dependencies and install if needed
 new.packages <- list.of.packages[!(list.of.packages %in% installed.packages()[,"Package"])]
-if(length(new.packages)) install.packages(new.packages)
-
+if(length(new.packages)) install.packages(new.packages, dependencies=c("Depends", "Imports",
+                                                                       "LinkingTo"))
+version_requirements <- c("3.3.2")
+if (!packageVersion("ggplot2") >= version_requirements[1]) {
```

Comment:
Why not `if (packageVersion("ggplot2") < version_requirements[1]) {`

Reply:
Same thing correct?  I guess at this point I dont want to have to go back and re-edit every file

Reply:
It is in the end.  I had to stop and think why you `!packageVersion("ggplot2")`. same same only different.

Reply:
Yeah....not sure why i went that way instead of the other way?!?!  I think I am just tired...